src/templates/plugin.utils.js

```diff
@@ -216,6 +216,40 @@ export function mergeAdditionalMessages (i18n, additionalMessages, localeCodes,
   }
 }
 
+/**
+ * @param {string} pathString
+ * @param {readonly string[]} localeCodes
+ * @param {import('../../types').DisableDefaultRedirect} disableDefaultRedirect
+ * @return {boolean}
+ */
+export function isEnabledRedirectForPath (pathString, localeCodes, disableDefaultRedirect) {
+  if (!disableDefaultRedirect) {
+    return true
+  }
+
+  if (typeof disableDefaultRedirect === 'boolean') {
+    return false
+  }
+
+  const clearPath = clearPathFromLocale(pathString, localeCodes)
+  const clearPathWithSlash = addSlashToPath(clearPath)
+
+  return !disableDefaultRedirect
+    .map((patternString) => {
+      const isPattern = patternString.includes('*')
```

Comment:
Looks like `*` is only supported at the end so you could use `endsWith` here.

Reply:
Currently i abandon the route array cuz i can't find it useful in the current implementation